File: 6.c
#include <stdio.h>

int main() 
{
    double num;
    printf("Enter a number: ");
    scanf("%lf", &num);
    if (num > 0) 
    {
        printf("positive\n");
    }
    else 
    {
        printf("negative\n");
    }
    return 0;
}

File: 6.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});
rl.question("Enter a number: ", function(input) {
  const num = parseFloat(input);

  if (num > 0) {
    console.log("positive");
  } else {
    console.log("negative");
  }
  rl.close();
});